tests/test_queries.py

```diff
@@ -16,86 +14,113 @@
 turing_paper = Paper.from_bibentry(fixtures.turing_bibentry,
                                    metadata=fixtures.turing_metadata)
 
+
 class TestAuthorFilter(unittest.TestCase):
 
     def test_fails_if_no_author(self):
         no_doe = doe_paper.deepcopy()
         no_doe.bibentry['author'] = []
-        self.assertTrue(not _check_author_match(no_doe, 'whatever'))
+        self.assertTrue(not AuthorFilter('whatever')(no_doe))
 
     def test_match_case(self):
-        self.assertTrue(_check_author_match(doe_paper, 'doe'))
-        self.assertTrue(_check_author_match(doe_paper, 'doe',
-                                            case_sensitive=False))
+        self.assertTrue(AuthorFilter('doe')(doe_paper))
+        self.assertTrue(AuthorFilter('doe', case_sensitive=False)(doe_paper))
+        self.assertTrue(AuthorFilter('Doe')(doe_paper))
 
     def test_do_not_match_case(self):
-        self.assertFalse(_check_author_match(doe_paper, 'dOe'))
-        self.assertFalse(_check_author_match(doe_paper, 'doe',
-                                             case_sensitive=True))
+        self.assertFalse(AuthorFilter('dOe')(doe_paper))
+        self.assertFalse(AuthorFilter('dOe', case_sensitive=True)(doe_paper))
+        self.assertFalse(AuthorFilter('doe', case_sensitive=True)(doe_paper))
+        self.assertTrue(AuthorFilter('dOe', case_sensitive=False)(doe_paper))
 
     def test_match_not_first_author(self):
-        self.assertTrue(_check_author_match(page_paper, 'motwani'))
+        self.assertTrue(AuthorFilter('motwani')(page_paper))
 
     def test_do_not_match_first_name(self):
-        self.assertTrue(not _check_author_match(page_paper, 'larry'))
+        self.assertTrue(not AuthorFilter('larry')(page_paper))
```

Comment:
Same comment regarding `assertFalse`/`assertIs(..., False)` instead of `assertTrue(not ...)`.  

Also, the test should probably be `lawrence` since `larry` isn't in the corresponding bibdata.